### ISSUE TITLE
Missing file(s)

### ISSUE THREAD

**Issue by gjamesgoenawan** [ucaszyp/MonoOcc#3]
Thank you for sharing your code! Really appreciate the effort to make your research available to the community.

I do have a problem regarding the KITTI dataloader, specifically in the file `projects/mmdet3d_plugin/datasets/semantic_kitti_dataset_stage2.py` in line 429, where there is a reference to a folder called `bev_feat_15.02`. I searched throughout your repository and couldn't find any reference to this folder. Would you mind explaining what the significance of it and how should I obtain it?

Furthermore, throughout this file, there are multiple absolute path reference (e.g. "/home/aidrive/") which leads to an error on my side. Would really appreciate it if you can clear it up! 

Thank You!

**Comment by xupeng-fu** [comment]
I have same question. Do you solve it ?

**Comment by VitaLemonTea1** [comment]
same to u

**Comment by ucaszyp** [comment]
We use the BEV features trained on multiview frames (privilage branch) to distill the monocular branch. You can train the privilaged branch by yourself to generate the intermediate BEV features, or you can load the weights of the privilaged branch and obtain BEV features to distill the single-frame network.

**Comment by VitaLemonTea1** [comment]
so it means we should train the privilaged branch first and get the bev_features?

**Comment by ucaszyp** [comment]
> so it means we should train the privilaged branch first and get the bev_features?

Yes, I will upload the pretrained weights of the privileged branch to the repository today. Joint training is limited by cuda memory. This work was done during the cvpr challenge last year, and its technology is now outdated. There are some lightweight and high-performance works now. If you want to find an existing technology as your code base first, I recommend CGFormer (https://github.com/pkqbajng/CGFormer).
Thank you for your interest in our work!!!

**Comment by VitaLemonTea1** [comment]
Thanks for your reply!
